StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Intent.swift

```diff
@@ -21,7 +21,7 @@ import UIKit
 enum Intent {
     case paymentIntent(STPPaymentIntent)
     case setupIntent(STPSetupIntent)
-    case deferredIntent(STPElementsSession)
+    case deferredIntent(elementsSession: STPElementsSession, intentConfig: PaymentSheet.IntentConfiguration)
```

Comment:
Updated this to also contain an intent config

Reply:
Nice, yep.